stdsimd/arch/detect/linux/auxvec.rs

```diff
@@ -0,0 +1,290 @@
+//! Parses ELF auxiliary vectors.
+
+use core::mem;
+use _std::prelude::v1::*;
+use _std::fs::File;
+use _std::io::Read;
```

Comment:
This unfortunately will cause some problems when building in libstd itself as this doens't exist, but could this continue to use the same strategy as before?